LogicLayer/Friends/Dtos/FriendshipStatusExtension.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using GameBoard.DataLayer.Enums;
+
+// I know that it is not the best namespace to put this in, but it did not work in other ones.
+
+namespace GameBoard.LogicLayer.Friends.Dtos
+{
+    public static class FriendshipsStatusExtensions
+    {
+        public static FriendRequestDto.FriendRequestStatus ToFriendRequest(this FriendshipStatus f)
```

Comment:
Change `f` to `friendshipStatus`, it doesn't hurt here but let's not make such names a common practice.

Reply:
Fixed.